src/runtime/vendor/github.com/kata-containers/govmm/qemu/qmp.go

```diff
@@ -775,7 +775,7 @@ func (q *QMP) blockdevAddBaseArgs(device, blockdevID string, ro bool) (map[strin
 		"driver":    "raw",
 		"read-only": ro,
 		"file": map[string]interface{}{
-			"driver":   "file",
+			"driver":   "host_device",
```

Comment:
Did the API change?

Reply:
Not really.   `host_device` has been the preferred way to do this since QEMU 3.0, but doing if via `file` was finally removed in QEMU 6.0.